osh/hub/scan/fixtures/initial_data.json

```diff
@@ -2724,5 +2724,21 @@
       1
     ]
   }
+},
+  {
+  "pk": 1,
+  "model": "scan.retentionpolicysetting",
+  "fields": {
+    "category": "PersonalScan",
+    "days": 7
```

Comment:
I would set the retention to 30 days by default.

Reply:
Done